packages/dds/tree/src/feature-libraries/flex-tree/flexTreeTypes.ts

```diff
@@ -248,6 +248,18 @@ export interface FlexTreeField extends FlexTreeEntity<FlexFieldSchema> {
 	 * Implementing this will require some care to preserve lazy-ness and work efficiently (without leaks) for empty fields, particularly on MapNodes.
 	 */
 	isSameAs(other: FlexTreeField): boolean;
+
+	/**
+	 * Gets a node of this field by its index without unboxing.
+	 * @param index - Zero-based index of the item to retrieve. Negative values are interpreted from the end of the sequence.
+	 *
+	 * @returns The element in the sequence matching the given index. Always returns undefined if index \< -sequence.length
+	 * or index \>= array.length.
```

Comment:
```suggestion
	 * or index \>= sequence.length.
```

Or "array" in both places - as long as it's consistent. Right now it says "sequence" for the less-than case and "array" for the greater-than case.